src/test/java/io/confluent/connect/jdbc/sink/integration/PostgresDatatypeIT.java

```diff
@@ -133,14 +230,35 @@ private void createTableWithUuidColumns() throws SQLException {
     log.info("Created table {} with UUID column", tableName);
   }
 
-  private void startTask() {
-    task = new JdbcSinkTask();
-    task.start(props);
+  private void createTableWithLessFields() throws SQLException {
+    log.info("Creating table {} with UUID column", tableName);
```

Comment:
nit: Please update this log line to be easier to distinguish from the method above. Ditto for the log line after table is created.